README.md

```diff
@@ -88,12 +88,13 @@ This script uses hershey stroke fonts to place text in Remarkable lines files wi
 
 Currently, one should initialize their connection to Remarkable Cloud using rmapy directly. You can obtain a one time token from [Remarkable's Mobile Connect](https://my.remarkable.com/connect/mobile) page.
 
-Then follow example as [described here](https://rmapy.readthedocs.io/en/latest/quickstart.html#registering-the-api-client). Eg,
+The official Wiki for rmapy has obsolete syntax, but may still prove useful as examples to follow if you're lost.[You can find it here.](https://rmapy.readthedocs.io/en/latest/quickstart.html#registering-the-api-client). Eg,
 
 ```python
 from rmapy.api import Client
 rmapy = Client()
 rmapy.register_device("<YOUR ONE TIME TOKEN>")
+rmapy.renew_token()
```

Comment:
This shouldn't be necessary as it's already renewed in rmcloud.py https://github.com/bsdz/remarkable-layers/blob/f96a73a4c6da2ce93ec42f21e7964a77a910af63/rmlines/rmcloud.py#L18

Reply:
Actually, I don't think this ever gets executed - for me atleast, the exception right above is raised and execution is stopped, so rmapy.renew_token() never gets called.